x-pack/test/functional/apps/canvas/smoke_test.js

```diff
@@ -12,7 +12,7 @@ export default function canvasSmokeTest({ getService, getPageObjects }) {
   const testSubjects = getService('testSubjects');
   const browser = getService('browser');
   const retry = getService('retry');
-  const PageObjects = getPageObjects(['common']);
+  const PageObjects = getPageObjects(['common', 'security']);
```

Comment:
nit: Doesn't looks like this adds any security PageObjects in the tests here, `security` can be removed